src/UnitTests/Bug/AssignableCollectionBug.cs

```diff
@@ -38,7 +38,7 @@ protected override void Because_of()
             [Fact]
             public void Should_copy_array()
             {
-                _dest.ImageData.ShouldBeSameAs(_source.ImageData);
+                _dest.ImageData.ShouldEqual(_source.ImageData);

```

Comment:
Array lengths are immutable, but their contents are not. The original
version of this did an array copy, I don't know why I copied the reference
over. This makes it consistent with all we other enumerables.

On Tuesday, June 28, 2016, Lucian Bargaoanu <notifications@github.com>
wrote:

> In src/UnitTests/Bug/AssignableCollectionBug.cs
> <https://github.com/AutoMapper/AutoMapper/pull/1376#discussion_r68718450>:
>
> > @@ -38,7 +38,7 @@ protected override void Because_of()
> >              [Fact]
> >              public void Should_copy_array()
> >              {
> > -                _dest.ImageData.ShouldBeSameAs(_source.ImageData);
> > +                _dest.ImageData.ShouldEqual(_source.ImageData);
>
> This seems wrong. Why copy? It could be 1MB.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/AutoMapper/AutoMapper/pull/1376/files/6afb6a88b27147c1d838f7ee9218242c1cf05cc3#r68718450>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAGYMlTMuTUFqlGq5bs5q4OKeh_HFGogks5qQN4qgaJpZM4I_WZj>
> .
>